app/views/docs/view.html.erb

```diff
@@ -1,2 +1,51 @@
+<% content_for(:title) { doc_title(@section,@document) } %>
 
-<%= raw @result %>
+<section class="docs-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto border-bottom">
```

Comment:
Cloud hosting should be green

![image](https://user-images.githubusercontent.com/8007722/71943990-6e12d100-3190-11ea-8783-1cdcf6519ce6.png)